native/screens/Roulette.jsx

```diff
@@ -56,7 +57,7 @@ const ProjectInfo = ({ project }) => {
       <View className="roulette-tools">
         <Text> Built with:</Text>
         {project.tools?.map((tool) => (
-          <Text>{tool}</Text>
+          <Text key={uuid.v4()}>{tool.name}</Text>
         ))}
       </View>
       <Text>{`Looking for collaborators who can commit at least X hours per week`}</Text>
```

Comment:
The number of hours could be added dynamically based on the `time_commitment` property; I used a switch statement in the SingleProject screen to add the number of hours that should work here, too.

Reply:
True! I'll add it to Jira